### PR TITLE
Change navbar hover color & box-shadow on scroll

### PR DESCRIPTION
**Pull request by kawaljeet99**
1. Changed the navbar link hover state & active state color to websites primary color i.e blue (On scrolled navbar only)
2. Removed the line under the active link from navbar (on scroll only)
3. Also changed the box-shadow value of the navbar for a better look on scroll which previously looked like a border-bottom.

[Before]
beforeEdits

[After]
after

### CODE REVIEW

**Comment by mehabhalodiya** [comment]
Please sign CLA.
For more about it, check [#31565 comment](https://github.com/kubernetes/website/pull/31565#issuecomment-1025113390)

**Comment by kawaljeet99** [comment]
I signed it

File: assets/scss/_custom.scss [hunk 1]
```diff
@@ -171,7 +171,7 @@ body.td-404 main .error-details {
 // Flip-Nav
 .flip-nav .td-navbar {
   background-color: white !important;
-  box-shadow: 0 1px 2px $medium-grey;
+  box-shadow: 0 1px 8px 0px #4c4c4c3d;
```
Comment (sftim) [inline assets/scss/_custom.scss:174]: Hi @kawaljeet99 

Would it be possible to use one of the theme's named colors here?

**Review by sftim — COMMENTED** [review]
Thanks for working on this.

One query about making the CSS maintainable (by using named colors); otherwise, I do like the intent and the implementation.

**Comment by sftim** [comment]
I'm overall not sure on whether we want the drop shadow _look_. I suspect the use of a CSS drop shadow attribute might have been to get a border but without some of the side effects of setting a border.

File: assets/scss/_custom.scss (hunk 1 above)
Comment (kawaljeet99) [reply assets/scss/_custom.scss]: Hi @sftim 

I have tweaked some values and achieved the similar look.
And as you asked, I've done this with named color from the theme.

**Comment by sftim** [comment]
Changes _work_.  I'm still unconvinced that the original intent of the `drop-shadow` was actually to set a drop shadow. Maybe it was intended as an alternative border, similar to https://css-tricks.com/snippets/css/multiple-borders/#aa-using-box-shadow

**Comment by kawaljeet99** [comment]
I understood your point that the intent of the `box-shadow` property used here was not to get a shadow but to get a `border` But the border look seems pretty old these days. So I would love to contribute more on this website (specifically towards improving the UI part) i.e. getting more modern look

**Comment by kawaljeet99** [comment]
> I understood your point that the intent of the `box-shadow` property used here was not to get a shadow but to get a `border` But the border look seems pretty old these days. So I would love to contribute more on this website (specifically towards improving the UI part) i.e. getting more modern look

What's your view on this @sftim ?

**Comment by tengqm** [comment]
@kawaljeet99 Thanks for the contribution. I don't think removing the shadow/border bar is improving the UI experience. We may want to consider people who are not good at telling the difference between colors.

**Comment by sftim** [comment]
I don't agree with changing the look towards a drop shadow look. The rest of the site doesn't use any skeumorphism (shadows, etc) in design. However we implement the border, it should continue to look like a flat border.

**Comment by tengqm** [comment]
A little bit clarification. By "border/shadow", I'm referring to the dark gray line under the word "Kubernetes Blog" in the first screenshot. Are we talking about the same thing? I prefer leave that as is. Switching to color based highlight is making me uncomfortable. I'm not a UI expert though.

**Comment by sftim** [comment]
> By "border/shadow", I'm referring to the dark gray line under the word "Kubernetes Blog" in the first screenshot. Are we talking about the same thing? I prefer leave that as is. 

That's also how I feel

Enhancing / altering the highlight for the top navigation bar? That could work. We'd need to make sure that however we do it, it works even with a banner displaying (there's one up now if folks want to see what I mean).

@kawaljeet99 I recommend using an _issue_ to discuss **what change we should make**; you can also use a PR to propose an implementation of that change, and for discussion about **whether the implementation matches the intended outcome**. Does that make sense?

**Comment by divya-mohan0209** [comment]
Hello @kawaljeet99, per Tim & Qiming's comments above I'll be closing this PR. Kindly open an issue first to discuss what changes need to be made & then a PR to propose an implementation of that change, and for discussion about whether the implementation matches the intended outcome. Kindly let us know if you have any questions.

**Pull request closed without merging**